### PR TITLE
Add order newest and oldest functionality in personality-page

### PR DESCRIPTION
**Pull request by pepermao**
image

image

image

https://www.figma.com/proto/J6OY3ZQILdR7H1qCWyqVw7/Aletheia?node-id=752%3A2107&scaling=min-zoom
close #348

### CODE REVIEW

File: src/components/List/SortBySelect.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+interface ISortByButton {
+  sortBy: string
+  refreshListItems: (sortBy: string) => void
+}
+const SortByButton = ({sortBy, refreshListItems}: ISortByButton) => {
+  const [visible, setVisible] = useState(false)
+  const [ value, setValue ] = useState('asc')
+
+  const handleOk = () => {
+      refreshListItems(value)
```
Comment (thesocialdev) [inline src/components/List/SortBySelect.tsx:14]: Let's change `refreshListItems` for  `onChange`, in this component only.

File: src/components/Modal/orderModal.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react"
```
Comment (thesocialdev) [inline src/components/Modal/orderModal.tsx:1]: The first letter should be capital in the name of the component.

File: src/components/Modal/orderModal.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    setVisible(false);
+  };
+
+  return (
+    <>
+      <Button type="primary" onClick={showModal}>
+        Open Modal with async logic
+      </Button>
+      <Modal
+        title="Title"
```
Comment (thesocialdev) [inline src/components/Modal/orderModal.tsx:33]: When you start adding styles to the Modal, move it to another component `BaseModal`. Se we can use it in other places.

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: src/components/List/SortBySelect.tsx (hunk 1 above)
Comment (pepermao) [reply src/components/List/SortBySelect.tsx]: the name onChange is confused with native function

File: src/components/Radio/orderRadio.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    return(
+        <Radio.Group
+            onChange={onChangeRadio}
+            value={value}
+            style={{
+                fontSize: '30px',
+            }}
+        >
+            <Space direction="vertical">
+                <Radio
```
Comment (thesocialdev) [inline src/components/Radio/orderRadio.tsx:18]: We can remove the ones that are no supported in the server-side, we only support `asc` and `desc`

File: src/styles/app.css [hunk 5]
```diff
@@ -3,3 +3,76 @@ body {
     padding: 0;
     margin: 0;
 }
+
+div.ant-modal-content {
+    border-radius: 30px;
```
Comment (thesocialdev) [inline src/styles/app.css:8]: Let's try to move CSS to inline as much as possible and avoid css files.

If you need css, use styled-components

File: src/components/List/SortByButton.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+const SortByButton = ({ refreshListItems }) => {
+    const [visible, setVisible] = useState(false)
+    const [ value, setValue ] = useState('asc')
+
+    return (
+        <>
+            <Button
+                shape="round"
+                icon={
+                    <SortAscendingOutlined style={{
```
Comment (thesocialdev) [inline src/components/List/SortByButton.tsx:15]: Let's enforce the black color here. Please use the colors lib for it.

File: src/components/List/SortByButton.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                        fontSize: '16px',
+                        fontWeight: 900,
+                        }}
+                    />
+                }
+                onClick={() => setVisible(!visible)}
+                style={{
+                    borderWidth: "2px",
+                    borderColor: '#515151',
```
Comment (thesocialdev) [inline src/components/List/SortByButton.tsx:24]: Let's use the colors lib instead of hardcoding the hex code

File: src/api/personality.ts [hunk 8]
```diff
@@ -7,7 +7,7 @@ const getPersonalities = (options = {}, dispatch) => {
         page: options.page ? options.page - 1 : 0,
         order: options.order || 'asc',
         name: options.searchName,
-        pageSize: options.pageSize ? options.pageSize : 5,
+        pageSize: options.pageSize,
```
Comment (thesocialdev) [inline src/api/personality.ts:10]: Are you sure removing the default option won't break anything?

**Review by thesocialdev — CHANGES_REQUESTED** [review]

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**